README.md

```diff
@@ -43,22 +43,21 @@ $ ember exam --load-balance --parallel=2 --server --no-launch
 $ ember exam --load-balance --parallel --server --no-launch
 ```
 
-The idea is that you should be able to replace `ember test` with `ember exam` and never look back.
+The idea is that you can replace `ember test` with `ember exam` and never look back.
 
-However, to get the unique features of Ember Exam (described in-depth below), you will need to import `addon-test-support/start.js` to invoke the `start` function. The start function creates an instance of EmberExamTestLoader, loads tests, and invokes start function from either `Ember-Qunit` or `Ember-Mocha`. This is done usually from within `test-helper.js` and should replace the use of `start()` from `Ember-Qunit` or `Ember-Mocha`.
+To get the unique features of Ember Exam (described in-depth below), you will need to **replace** the use of `start()` from `Ember-Qunit` or `Ember-Mocha` in `test-helper.js` with `start()` from `ember-exam`:
```

Comment:
since `start()` from `ember-exam/test-support/start` should replace both `loadEmberExam()` from `ember-exam/test-support/load' and `start()` from `Ember-Qunit` or `Ember-Mocha`. (note, start() from ember-exam will already call loadEmberExam() internally)
Should we call out that `loadEmberExam()` is also no longer needed to be extra clear?